src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ayamiss.cpp

```diff
@@ -379,8 +337,7 @@ class spell_ayamiss_swarmer_start_loop : public SpellScript
 
     void HandleScript(SpellEffIndex /*effIndex*/)
     {
-        uint32 loopSpells[3] = { SPELL_HIVEZARA_SWARMER_LOOP_1, SPELL_HIVEZARA_SWARMER_LOOP_2, SPELL_HIVEZARA_SWARMER_LOOP_3 };
-        GetCaster()->CastSpell((Unit*)nullptr, Acore::Containers::SelectRandomContainerElement(loopSpells));
+        GetCaster()->GetAI()->DoCastAOE(RAND(SPELL_HZ_SWARMER_LOOP_1, SPELL_HZ_SWARMER_LOOP_2, SPELL_HZ_SWARMER_LOOP_3));
```

Comment:
Can GetAI() return nullptr?

Reply:
Can very early during object creation. Will never be the case here.